clinica/utils/input_files.py

```diff
@@ -401,24 +401,49 @@ def t1_volume_native_tpm_in_mni(tissue_number, modulation):
     }
 
 
-def t1_volume_template_tpm_in_mni(group_label, tissue_number, modulation):
+def t1_volume_template_tpm_in_mni(group_label, tissue_number, modulation, fwhm=None):
```

Comment:
Just to be sure, can `fwhm` be set to None in click (since the default is 8) ?

Reply:
I implemented it this way for backward compatibility concerns.

I was however wondering the same thing. Indeed, when you look at the code of Dartel2MNI, especially here:

https://github.com/aramis-lab/clinica/blob/02aaa90321ec853fb09ea2f965b1b338921a1845/clinica/pipelines/t1_volume_dartel2mni/t1_volume_dartel2mni_pipeline.py#L265-L268

It feels like you *could* intentionally ask for no smoothing by not setting `fwhm` in the CLI, but this is not what's currently happening since there is a default value implemented both in the CLI and in the `check_parameters` function. Unless I missed something, there is currently no way (available to the user) to have the smoothing parameter equal to None at this point.

I agree that this is very confusing. I think that if smoothing is required, then we should remove the condition in the workflow, otherwise, we should have no smoothing if the user didn't pass explicitly a value for fwhm in the CLI. The docs could also be clarified to reflect and stress this point.

Reply:
I agree. Based on the docs, I'd expect the stats to be computed on smoothed volumes only if I provide a value for `fwhm` in the CLI. If nothing is provided for `fwhm`, it feels very weird to have the stats computed on 8mm-smoothed images.

Reply:
After looking into it, I reverted the modifications I initially made to `t1_volume_parcellation`, leaving the current behavior in place. I only changed the `statistics_volume` pipeline to use the `fwhm` value provided (which should match the behavior for pet).